playbook/app/pb_kits/playbook/pb_table/docs/_sticky_table_sm.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import Table from '../_table'
+
+const StickyTableSm = (props) => {
```

Comment:
I would removed `sm` from StickyTable name since sticky-header is not related to the table size.

Reply:
I agree, i am fixing this now. Thank you!